core/src/typecheck/error.rs

```diff
@@ -146,7 +319,7 @@ impl UnifError {
     }
 
     /// Convert a unification error to a typechecking error, given a populated [name
-    /// registry][reporting::NameReg]. Actual meat of the implementation of
+    /// registry][reporting::NameReg]. inferred meat of the implementation of
```

Comment:
```suggestion
    /// registry][reporting::NameReg]. Actual meat of the implementation of
```

Reply:
Ah, (too) wild search and replace leads to fun results :sweat_smile: